src/main/kotlin/no/nav/syfo/behandler/BehandlerService.kt

```diff
@@ -78,13 +78,19 @@ class BehandlerService(
             .map { (pBehandler, _) -> pBehandler.id }.firstOrNull() != pBehandler.id
 
         val behandlerIkkeKnyttetTilArbeidstaker = !pBehandlereForArbeidstakerList
+            .filter { (_, behandlerType) -> behandlerType == behandlerArbeidstakerRelasjon.type.name }
```

Comment:
Tilpasset slik at vi legger inn en relasjon til legen som sykmelder selv om den finnes fra før som fastlege. Vi ble enige om at dette er mer konsistent siden det vil være sånn fra før at det ble to relasjoner dersom sykmeldingen kommer først.